go-selinux/selinux_linux.go

```diff
@@ -410,6 +427,13 @@ func CanonicalizeContext(val string) (string, error) {
 	return readWriteCon(filepath.Join(getSelinuxMountPoint(), "context"), val)
 }
 
+/*
+CreateContext requests the type transition from source to target for class  from the kernel.
```

Comment:
This seems badly named.

Reply:
What name would you prefer? ComputeCreateContext?